components/skus/browser/rs/cxx/src/lib.rs

```diff
@@ -414,16 +393,14 @@ impl CppSDK {
 
     fn submit_receipt(
         self: &CppSDK,
-        callback: SubmitReceiptCallback,
-        callback_state: UniquePtr<ffi::SubmitReceiptCallbackState>,
+        callback_state: UniquePtr<ffi::RustBoundPostTask>,
```

Comment:
shouldn't we keep the param name `callback` instead of `callback_state`?

Reply:
Fixed.